.github/workflows/travis.yml

```
@@ -0,0 +1,23 @@
+name: Test
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened, ready_for_review, labeled, unlabeled]
```

```suggestion
    types: [opened, synchronize, reopened, labeled, unlabeled]
```
I don't like when the tests are restarted on `ready_for_review`, it is waste of time.